### DISCUSSION TITLE
"Recommended based on people you follow"

### DISCUSSION THREAD

**Discussion by ozanvos** [community/community#54329] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I see people recommended on my feed with the explanation "Recommended based on people you follow", but clearly aren't related to any GitHub users I follow. They are from a previous company that had a pretty toxic work environment and just seeing those people brings out bad memories. I want to turn this off and not see these people every time I open GitHub.

**Comment by zakjanzi** [comment]
Agreed. Like why does everyone need to know what repos I star or create? It's just unnecessary.

**Comment by Ant-f** [comment]
Agreed. The option to exclude/block users might be nice too.
